src/main/java/de/zalando/aruha/nakadi/repository/zookeeper/ZooKeeperHolder.java

```diff
@@ -55,13 +53,7 @@ public void init() throws Exception {
         EnsembleProvider ensembleProvider;
         if (exhibitorAddresses != null) {
             final Collection<String> exhibitorHosts = Arrays.asList(exhibitorAddresses.split("\\s*,\\s*"));
-            final Exhibitors exhibitors = new Exhibitors(exhibitorHosts, exhibitorPort,
-                    new BackupConnectionStringProvider() {
-                        @Override
-                        public String getBackupConnectionString() throws Exception {
-                            return zookeeperBrokers + zookeeperKafkaNamespace;
-                        }
-                    });
+            final Exhibitors exhibitors = new Exhibitors(exhibitorHosts, exhibitorPort, () -> zookeeperBrokers + zookeeperKafkaNamespace);
```

Comment:
I did such a refactoring in this place a couple of times. But then my branch was never merged to "master" branch. So this is a really bad sign to make this refactoring :laughing: